__test__/homePage.test.js

```diff
@@ -0,0 +1,90 @@
+  //   }
+  //   testStatus = false;
+  // });
+  // // });
+  // it('should load homepage', () => {
+  //   // driver.get('https://www.trilliontreecampaign.org/').then(() => {
+  //     driver.getTitle().then((title) => {
+  //       expect(title).toBe("Plant trees around the world - Plant-for-the-Planet");
+  //       testStatus = true;
+  //       // driver.quit();
+  //     });
+  //   // });
+  // });
+  it('should click on donate', async () => {
+    // load().then(async () => {
+      // const val = await donateButton();
+      // await val.click();
+      await driver.wait(until.elementLocated(By.className('donateButton'))).click();
+      await driver.switchTo().activeElement();
+      await driver.wait(until.elementLocated(By.id('treeDonateContinue')), 20000).click();
+      await driver.findElement(By.name('firstName')).sendKeys('Bright');
+      await driver.findElement(By.name('lastName')).sendKeys('Amidiagbe');
+      await driver.findElement(By.name('email')).sendKeys('captainamiedi1@gmail.com');
+      await driver.findElement(By.name('address')).sendKeys('43 block');
+      await driver.findElement(By.name('city')).sendKeys('surulere');
+      const country = await driver.findElement(By.name('countrydropdown'));
+      await country.click();
+      await country.clear();
+      await country.sendKeys(Key.CONTROL + "a");;
+      await country.sendKeys(Key.DELETE);
+      await country.sendKeys('United States of America');
+      await country.sendKeys(Key.RETURN);
+      await driver.findElement(By.name('zipCode')).sendKeys('85001'); // for netherland 6176 ZG
+      const val1 = await driver.findElement(By.xpath("//*[text()='Continue']"));
+      await val1.click();
+      await driver.switchTo().activeElement();
+      (await driver).sleep(100);
+      // await driver.wait(until.elementLocated(By.xpath("//div[@id='cardNumber']/div/input"))).sendKeys('4242424242424242');
+      const cardNumber = await driver.wait(until.elementLocated(By.xpath("//div[@id='cardNumber']/div/input")), 10000);
+      const cardEnable = await driver.wait(until.elementIsEnabled(cardNumber));
+      (await driver).sleep(3000);
+      await cardEnable.sendKeys('424242424242424242');
+      // }
+      const cardNo = '4';
+      for (let i = 0; i < cardNo.length; i++) {
+        cardEnable.sendKeys(cardNo.charAt(i));
+      }
+      (await driver).sleep(100);
+      const expiry = await driver.wait(until.elementLocated(By.xpath("//div[@id='expiry']/div/input")), 10000);
+      const expiryDate = await driver.wait(until.elementIsEnabled(expiry));
+      const cardExpiry = '445';
+      for (let i = 0; i < cardExpiry.length; i++) {
+        expiryDate.sendKeys(cardExpiry.charAt(i));
+      }
+      await expiryDate.sendKeys('4444');
+      (await driver).sleep(100);
+      const cvc = await driver.wait(until.elementLocated(By.xpath("//div[@id='cvc']/div/input")), 10000);
+      await cvc.sendKeys('1111');
```

Comment:
Actually I do not understand this code. Why not just click on a field and send keys?

Reply:
Ok, filling data in these Stripe form fields (iframe) seems to be not always reliable. We must solve this problem first, before we can merge this PR. It's also not reliable in my [testing branch](https://github.com/Plant-for-the-Planet-org/planet-webapp/pull/701).

Reply:
for expiry maybe send eg: `12/24` with `/` present?

Reply:
If you are interested, have a look at the video of the test session. Sometimes the first number of the credit card is not inserted into the field. Sometimes the CVC code just get not entered into the CVC field. (Both resulting into invalid payment data - thus the donate button can not be pressed). @captainamiedi already tried tricks to make it work, but I am not sure if he already found a solution working all the time as I sometimes still got errors testing (locally).

Reply:
Maybe this helps?
https://stackoverflow.com/questions/52608566/selenium-send-keys-incorrect-order-in-stripe-credit-card-input

Reply:
I think I fixed this now (selenium needs to switch to the iframe before entering data and then switch back to the parent and the other iframes).